pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -134,6 +134,11 @@ def load(fnames, tag=None, sat_id=None, malformed_index=False):
     latitude = 90.0 * np.cos(angle)
     data['latitude'] = (('time'), latitude)
 
+    # create constant altitude at 400 km
+    alt0 = 400.0
+    altitude = alt0 * np.ones(data['latitude'].shape)
+    data['altitude'] = (('time'), altitude)
+
```

Comment:
So this whole bit, why not just add a single value as altitude to the coordinates?  `data.coords['altitude'] = alt0`.

Reply:
The values in the coordinates do not refer to the satellite altitude, but to the altitude of a remote sensing measurement (hence the 2d data).  It could definitely be clearer (see below).  I'm open to naming convention suggestions.

Reply:
I would use "profile_height", since this is the height from the instrument (I assume?) rather than the altitude from the surface of the earth.  I would still add altitude to the coordinates, since part of the point of xarray is to not assign a bunch of the same values that aren't changing.